src/auto_dev_loop/add_repo.py

```
@@ -0,0 +1,411 @@
+            timeout=10,
+        )
+    except subprocess.TimeoutExpired as exc:
+        raise AddRepoError("GitHub CLI timed out. Check your network connection.") from exc
+    except (FileNotFoundError, subprocess.CalledProcessError) as exc:
+        raise AddRepoError(
+            "GitHub CLI (gh) is not installed or not working. "
+            "Install from https://cli.github.com/"
+        ) from exc
+
+    result = subprocess.run(
+        ["gh", "auth", "status"],
```

Ruff (0.15.4)

[error] Starting a process with a partial executable path

(S607)